# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-auto-first-observation-immediate.html.ini
# Commit: 95679f6ce254
# Full Hash: 95679f6ce2544b21ec9cfd0d19961058d60155d6
# Author: Frederic Wang <fred.wang@free.fr>
# Date: 2023-11-27 17:00:06
# Regressor Bug: 1807253
# File Overlap Count: 1
# Description:
#   Bug 1807253 - unreliable timing for content-visibility:auto, r=emilio
#   
#   Rewrite implementation of content-visibility: auto as defined in
#   https://github.com/w3c/csswg-drafts/issues/8542
#   
# ==============================================================================

diff -r 9ea04a2cc829 -r 95679f6ce254 testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-auto-first-observation-immediate.html.ini
--- a/testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-auto-first-observation-immediate.html.ini	Mon Nov 27 08:00:41 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-[content-visibility-auto-first-observation-immediate.html]
-  [Target is sized and laid out before resize observer]
-    expected: FAIL